solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBHttp2SolrClient.java

```diff
@@ -138,8 +136,9 @@ public Set<String> getUrlParamNames() {
   public CompletableFuture<Rsp> requestAsync(Req req) {
     CompletableFuture<Rsp> apiFuture = new CompletableFuture<>();
     Rsp rsp = new Rsp();
-    boolean isNonRetryable =
-        req.request instanceof IsUpdateRequest || ADMIN_PATHS.contains(req.request.getPath());
+    boolean isAdmin =
```

Comment:
out of scope but ADMIN-ness doesn't actually relate to retry-ability.  Ideally the request would indicate this, and would have a default implementation that looks at HTTP GET vs other verbs.

Reply:
Yeah, IMO it makes sense to give `SolrRequest` a `isRetryable()` method or something similar that clients can key off of.  But very much out of scope here...